tests_metricflow/fixtures/dataflow_fixtures.py

```diff
@@ -90,7 +91,26 @@ def scd_query_parser(  # noqa: D103
 
 
 @pytest.fixture(scope="session")
-def time_spine_source(  # noqa: D103
+def time_spine_sources(  # noqa: D103
     sql_client: SqlClient, mf_test_configuration: MetricFlowTestConfiguration  # noqa: F811
-) -> TimeSpineSource:
-    return TimeSpineSource(schema_name=mf_test_configuration.mf_source_schema, table_name="mf_time_spine")
+) -> Mapping[TimeGranularity, TimeSpineSource]:
+    legacy_time_spine_grain = TimeGranularity.DAY
+    time_spine_base_table_name = "mf_time_spine"
+    print("expected schema name:", mf_test_configuration.mf_source_schema)
+    # Legacy time spine
+    time_spine_sources = {
+        legacy_time_spine_grain: TimeSpineSource(
+            schema_name=mf_test_configuration.mf_source_schema, table_name=time_spine_base_table_name
+        )
+    }
```

Comment:
Wait, does this mean that we're no longer triggering the fallback behavior in the runtime, because we're overriding it with an explicit time spine input?

What happens if I define an HOUR grain time spine but leave the DAY grain one in the original configuration? Does that fail unceremoniously, do we raise an informative error, or do we just use the HOUR spine for everything?

I'm actually fine with raising an informative error or using the HOUR spine, especially for now, I just realized I'm just not clear on what happens.

Reply:
The legacy time spine config will only be included in your manifest if you have the `metricflow_time_spine` model in your project. It will be overridden if you add configs pointing to a time spine with `DAY`. The legacy time spine will be treated like any other time spine you have configured.
If you have a new time spine with `HOUR` grain in addition to the legacy `DAY` time spine, we'll use the `DAY` time spine if you query something with `DAY` or higher. In that case we would only use the `HOUR` time spine if you query with `HOUR`. We'll choose the time spine with the largest grain available that can resolve the requested grain.